files/build_templates/sonic_debian_extension.j2

```diff
@@ -50,6 +52,10 @@ sudo chroot $FILESYSTEM_ROOT service docker start
 # Apply apt configuration files
 sudo cp $IMAGE_CONFIGS/apt/sources.list $FILESYSTEM_ROOT/etc/apt/
 sudo cp -R $IMAGE_CONFIGS/apt/sources.list.d/ $FILESYSTEM_ROOT/etc/apt/
```

Comment:
sudo cp $IMAGE_CONFIGS/apt/sources.list.d/$(CONFIGURED_ARCH)/ $FILESYSTEM_ROOT/etc/apt/

then, you can put the needed sources file in the arch directory.

Reply:
Done.